config/webpacker.yml

```diff
@@ -5,6 +5,7 @@ default: &default
   source_entry_path: packs
   public_output_path: packs
   cache_path: tmp/cache/webpacker
+  webpack_compile_output: true
```

Comment:
Is this related to this PR? 😱 What for? 🤔

Reply:
I added it because it is easier to debug when something fails in the JS.. but It shouldn't be here in the PR 😄 

will remove it